contracts/axelar-gas-service/src/test.rs

```diff
@@ -81,15 +159,60 @@ fn pay_gas_for_contract_call() {
     );
 }
 
+#[test]
+/// Ensure failure occurs when refund_amount <= 0
+fn fail_collect_fees_invalid_amount() {
```

Comment:
```suggestion
fn fail_collect_fees_zero_refund_amount() {
```
instead of adding comments, make the test name more descriptive